src/test/java/com/cburch/logisim/StartupTest.java

```diff
@@ -0,0 +1,199 @@
+      "-m x",                       // bad format, missing sizes
+      "-m 10x10+",                  // bad format, missing location
+      "--locale",                   // missing require parameter
+      "--locale wrong",             // sub-option does not exist
+      "--user-template",            // missing required parameter
+      "-u a -u b",                  // can't template twice
+
+      // TTY options
+      "--tty",                      // missing required parameter
+      "-t table a b",               // too many files for --tty
+      "-t table",                   // missing file
+      "--tty other-thing",          // sub-option does not exist
+      "--load",                     // missing required parameter
+      "--load file",                // missing --tty
+      "-l a -l b",                  // can't load twice
+      "--save",                     // missing required parameter
+      "--save file",                // missing --tty
+      "--save a --save b",          // can't save twice
+      "-s",                         // missing required parameters
+      "-s a",                       // missing required parameter
+      "-s a b -s a c",              // can't substitute the same thing twice
+      "--toplevel-circuit",         // missing required parameter
+      "--toplevel-circuit a",       // missing file
+      "--test-vector",              // missing required parameters
+      "--test-vector a",            // missing required parameter
+      "--test-vector a b",          // missing file
+      "--test-fpga",                // missing required parameters
+      "--test-fpga a",              // missing required parameter
+      "--test-fpga a b",            // missing file
+      "--test-fpga a b 1 d",        // too many files
+      "--test-fpga a b 1 HDLONLY d",// too many files
+      "--test-fpga a b HDLONLY 1 d",// too many files
+      "--test-fpga a b c d",        // too many files
+      "--test-circuit",             // missing required parameter
+      "--test-circuit a b",         // missing file
+      "--test-circuit a b c d",     // too many files
+      "--new-file-format",          // missing required parameters
+      "-n a",                       // missing required parameter
+      "-n a b c",                   // too many files
+
+      // TTY combinations
+      "-t table -n a b",            // multiple TTY tasks
+      "-t table -test-fpga a b c",  // multiple TTY tasks
+      "-t csv --test-vector a b",   // multiple TTY tasks
+      "-t tab --test-circuit a b",  // multiple TTY tasks
+
+      // TTY / GUI combinations
+      "-t tab -g ansi",
+      "-t tab -m9x9",
+      "-t tab -u a",
+      
+      "--does-not-exist"            // unknow parameter
+    );
+  }
+
+  /** Check various good TTY options. */
+  @ParameterizedTest
+  @MethodSource
+  public void testGoodTty(String options) {
+    String[] args = options.split(" ");
```

Comment:
No explicit type hints `var`/`final var` (multiple places).

Reply:
I do not know what you mean. Are you saying the Right way is to NOT specify the type and allow the compiler to deduce it?

Reply:
Yes, exactly. Please write `final var args = options.split(" ");` and compiler will deduct it's `String[]` because this is what `split()` return type is.

Reply:
done